partcad-cli/src/partcad_cli/click/commands/info.py

```
@@ -0,0 +1,65 @@
+    "--param",
+    "params",
+    type=str,
+    multiple=True,
+    metavar="<name>=<value>",
+    help="Assign a value to the parameter",
+)
+@click.argument("object", type=str, required=False)  # help="Part (default), assembly or scene to show"
+@click.pass_obj
+def cli(ctx, package, interface, assembly, sketch, scene, object, params):  # , path
+    params = {}
+    if not params is None:
```

Ruff (0.8.2)

Test for object identity should be `is not`

Convert to `is not`

(E714)